docs/content-server-api/deployments/get.md

```diff
@@ -90,6 +107,13 @@ List all deployments stored in the content server
   - Default value: 500
   - Description: The deployments are a paginated collection, this parameter corresponds to the limit for each page.
 
+- lastId
+  - Format: string
+  - Value: EntityId
+  - Example: lastId=QmNknKv8MuKbfZ73z4QdUEsNbTd1ZAN1fSuwTFGiNGeCt5
+  - Default value: -
+  - Description: It is the last entity id that will be listed in the collection
```

Comment:
If `lastId` is set, then it won't be listed at all, it should be avoided. Right? Or am I getting something wrong?